code/students-performance/src/test/java/ru/unn/agile/StudentsPerformance/Model/PerformanceCalculatorTest.java

```diff
@@ -0,0 +1,219 @@
+package ru.unn.agile.StudentsPerformance.Model;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class PerformanceCalculatorTest {
+
+    private static final double COMPARE_DELTA = 0.001;
+
+    @Test
+    public void canCreateCalculator() {
+        List<Student> students = generateDefaultStudentsList();
+        PerformanceCalculator calculator = new PerformanceCalculator(students);
+        assertNotNull(calculator);
+    }
+
+    @Test (expected = IllegalArgumentException.class)
+    public void canCreateCalculatorIfStudentsAreNull() {
+        List<Student> students = null;
+        new PerformanceCalculator(students);
+    }
+
+    @Test (expected = IllegalArgumentException.class)
+    public void canCreateCalculatorIfStudentsAreEmpty() {
```

Comment:
The same. Please correct the name of test.

Reply:
Done